### PR TITLE
default cropping leases end date to two days from now

### PR DESCRIPTION
**Pull request by akash1810**
request from Jonny

### CODE REVIEW

**Comment by paperboyo** [comment]
@akash1810 We have an 👍  from Joanna.

**Comment by akash1810** [comment]
this isn't limited to only cropping leases and it should be 😢 not as simple a change as I'd hoped!

File: kahuna/public/js/leases/leases.js [hunk 1]
```diff
@@ -223,6 +225,17 @@ leases.controller('LeasesCtrl', [
             ctrl.updateLeases();
         });
 
+        $scope.$watch(() => ctrl.access, onValChange(selectedLeaseType => {
+            const isCroppingLease = ['allow-use', 'deny-use'].includes(selectedLeaseType);
+
+            if (isCroppingLease) {
+                // cropping leases should expire in 2 days, by default
```
Comment (franziskas) [inline kahuna/public/js/leases/leases.js]: could this comment me a method name?

Comment (akash1810) [reply kahuna/public/js/leases/leases.js]: Sure! Something like this?

```js
function getDefaultLeaseExpiry() { return moment().add(2, 'days').startOf('day').toDate();}
```

**Comment by franziskas** [comment]
Looks great :)

**Pull request merged**